### PR TITLE
chore(conductor, composer)!: update chain_id strings to rollup_name

### PR DESCRIPTION
**Pull request by joroshiba**
## Summary
changed references to `chain_id` that were strings to `rollup_name` or `rollup_id` from the sequencer viewpoint.

## Background
Chain ID is hugely overloaded, we previously updated our protos, but we haven't updated some configuration to remove the overloading.

## Changes
- updated conductor config
- updated composer + comments in composer

## Breaking Changes
- changes to conductor configuration

### CODE REVIEW

File: crates/astria-composer/local.env.example [hunk 1]
```diff
@@ -7,8 +7,8 @@ ASTRIA_COMPOSER_API_LISTEN_ADDR="0.0.0.0:0"
 # Address of the RPC server for the sequencer chain
 ASTRIA_COMPOSER_SEQUENCER_URL="http://127.0.0.1:26657"
 
-# A list of execution <chain_id_1>::<rpc_server_1>,<chain_id_2>::<rpc_server_2>.
-# Chain IDs are not case sensitive. If an ID is repeated, the last list item is used.
+# A list of execution <rollup_name_1>::<rpc_server_1>,<rollup_name_id_2>::<rpc_server_2>.
+# Rollup Names are not case sensitive. If a name is repeated, the last list item is used.
```
Comment (SuperFluffy) [inline crates/astria-composer/local.env.example:11]:
```suggestion
# Rollup names are not case sensitive. If a name is repeated, the last list item is used.
```
Also maybe mention that the names will be lowercased.

**Review by SuperFluffy — APPROVED** [review]
Good follow-up to tie up loose ends after https://github.com/astriaorg/astria/pull/436 got merged.

File: crates/astria-composer/src/searcher/rollup.rs [hunk 2]
```diff
@@ -53,17 +53,17 @@ impl Rollup {
...
-            chain_id,
+            rollup_name,
             url,
         })
     }
 
     pub(super) fn into_parts(self) -> (String, String) {
         let Self {
-            chain_id,
+            rollup_name: chain_id,
```
Comment (noot) [inline crates/astria-composer/src/searcher/rollup.rs:66]:
```suggestion
            rollup_name,
```

File: crates/astria-composer/src/searcher/rollup.rs [hunk 3]
```diff
@@ -53,17 +53,17 @@ impl Rollup {
...
             url,
         } = self;
         (chain_id, url)
```
Comment (noot) [inline crates/astria-composer/src/searcher/rollup.rs:69]:
```suggestion
        (rollup_name, url)
```

File: crates/astria-composer/src/searcher/rollup.rs (hunk 2 above)
Comment (SuperFluffy) [reply crates/astria-composer/src/searcher/rollup.rs:66]: Good point actually. The rename here is silly.

File: crates/astria-composer/src/searcher/rollup.rs (hunk 3 above)
Comment (SuperFluffy) [inline crates/astria-composer/src/searcher/rollup.rs:69]: And with the change above:
```suggestion
        (rollup_name, url)
```

File: crates/astria-composer/src/searcher/mod.rs [hunk 4]
```diff
@@ -297,14 +297,14 @@ impl Searcher {
...
+                .map(|fut| (rollup_name.clone(), fut))
             })
             .collect::<FuturesUnordered<_>>();
-        while let Some((chain_id, maybe_err)) = statuses.next().await {
+        while let Some((rollup_name, maybe_err)) = statuses.next().await {
             if let Err(e) = maybe_err {
                 return Err(e).wrap_err_with(|| {
                     format!(
-                        "collector for chain ID {chain_id} failed while waiting for it to become \
+                        "collector for rollup {rollup_name} failed while waiting for it to become \
```
Comment (SuperFluffy) [inline crates/astria-composer/src/searcher/mod.rs:307]:
```suggestion
                        "collector for rollup `{rollup_name}` failed while waiting for it to become \
```
I like doing this to have the name stand out a bit more.

**Review by SuperFluffy — APPROVED** [review]
Minor suggestion. In general I think there are likely other places where our variable naming is not perfect with respect to chain Id, rollup ID, and name. But I think that's ok.

**Review by noot — APPROVED** [review]

**Review by sambukowski — APPROVED** [review]

**Comment by joroshiba** [comment]
Closing in favor of https://github.com/astriaorg/astria/pull/791 due to staleness and many changes since put together.

**Pull request closed without merging**